src/lib/main.js

```diff
@@ -88,6 +91,8 @@ import * as widgets from '../widgets/index.js';
  */
 const instantsearch = toFactory(InstantSearch);
 
+instantsearch.routing = routing;
```

Comment:
I thought (but can't find the discussion, might have been informal) that we would go for instantsearch.routers (since it's an object with the default routers we ship). Wdyt?

This would also differentiate it from instantsearch({routing})

Reply:
Yes indeed. 👍